tests/suites/test_suite_pkparse.data

```diff
@@ -106,6 +106,18 @@ Parse Public RSA Key #1 (PKCS#8 wrapped)
 depends_on:MBEDTLS_MD5_C:MBEDTLS_PEM_PARSE_C
 pk_parse_public_keyfile_rsa:"data_files/format_gen.pub":0
 
+Parse Public RSA Key #1 (PKCS#8 wrapped, DER)
+depends_on:MBEDTLS_MD5_C
```

Comment:
Can you create a test case that uses SHA-256 instead (either in replacement or in addition to that one)? So that modern configurations that don't have MD5 or SHA1 still have a test for that feature.

Reply:
sure, I converted the original `format_gen.pub` file to `pkcs1` , but I will create a new pair of `pkcs8` `PEM` and `DER`

Reply:
@mpg I have just looked in the code, and code itself requires PEM format encrypted only when `MBEDTLS_MD5_C` is defined, which is strange to me why there is a need for an `MD` algorithm for encrypted `PEM` in the first place..

Reply:
Nevermind, it uses `pem_pbkdf1` internally, which uses `MD5`

Reply:
@RonEld That's because there's a PBKDF in between the PEM password and the actual encryption key used (it's a deliberately expensive function to slow down dictionary attacks).

Reply:
@hanno-arm yes, I saw it after posting.